src/layup/utilities/file_io/file_output.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 
 
-def write_csv(data, filepath):
+def write_csv(data, filepath, move_columns={}):
```

Comment:
Also I think it's better to not have an empty array as the default value. Better to use None instead of {} https://www.codiga.io/blog/python-avoid-empty-array-parameter/